data/sql/updates/pending_db_world/SL_gem_loot_fix.sql

```diff
@@ -0,0 +1,2 @@
+
+UPDATE `acore_world`.`reference_loot_template` SET `Item` = '30559', `Comment` = 'Etched Fire Opal' WHERE (`Entry` = '43010') and (`Item` = '30556');
```

Comment:
remove `acore_world` and the quotes.

see https://www.azerothcore.org/wiki/sql-standards for SQL standards